aten/src/ATen/native/NamedTensor.cpp

```diff
@@ -16,5 +16,144 @@ Tensor set_names(const Tensor& self, optional<DimnameList> names) {
   return result;
 }
 
+static void report_moving_unnamed_dim_error(
+    DimnameList names, DimnameList other, bool is_aligning_two_tensors) {
+  if (is_aligning_two_tensors) {
+    TORCH_CHECK(false,
+        "Could not infer the alignment of Tensor", names, " and Tensor", other,
+        ", please name the unnamed dimensions to resolve ambiguity.");
+    TORCH_CHECK(false,
```

Comment:
is this reachable?

Reply:
Oops I think this was a copy-paste fiasco